src/main/java/com/synopsys/integration/alert/provider/blackduck/collector/BlackDuckPolicyMessageContentCollector.java

```diff
@@ -24,95 +24,41 @@
 
 package com.synopsys.integration.alert.provider.blackduck.collector;
```

Comment:
The classes in the collector package all start with BlackDuck<NOTIFICATIONTYPE>MessageContentCollector for names.  I am thinking of simplifying the name to <NOTIFICATIONTYPE>Collector.  Any thoughts from the team?

Reply:
Some of my comments were omitted. BlackDuck**_NOTIFICATIONTYPE_**MessageContentCollector are the current names.  I was thinking of simplifying them to **_NOTIFICATIONTYPE_**Collector but wanted your thoughts on that first.

Reply:
I like BlackDuck**NOTIFICATIONTYPE**Collector. We should include the provider name for now in case providers have collisions in notification type names.